crates/re_space_view_spatial/src/scene/contexts/non_interactive_entities.rs

```diff
@@ -0,0 +1,34 @@
+use nohash_hasher::IntSet;
+use re_log_types::EntityPathHash;
+use re_viewer_context::SceneContextPart;
+
+/// List of all non-interactive entities for lookup during picking evaluation.
```

Comment:
Interesting... this does feel a bit awkward. Need to think about how to clean it up when moving this over to the blueprint.

Reply:
yes, this one is weird. We should solve this differently. This was a shortcut I took because I couldn't figure how to do that better. I mean really if the picking code could just do a blueprint query this would be nicer!!
I'll leave a todo note with both our handles ;)